packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -172,6 +173,21 @@ class VMService {
       });
     }
 
+    _peer.registerMethod('getFlutterVersion', (rpc.Parameters params) async {
```

Comment:
I would call this `flutterVersion` instead of `getFlutterVersion` to match the Dart style better.

Reply:
done.